.github/workflows/cd.yml

```diff
@@ -20,10 +20,17 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: 1.8
+    - name: download heroku cli
+      uses: wei/curl@master
+      with:
+        args: https://cli-assets.heroku.com/install.sh >> ./.github/heroku_install.sh
+      - name: Install heroku cli
+        run: |
+          sudo chmod u+x ./.github/heroku_install.sh
+          sudo bash ./.github/heroku_install.sh
```

Comment:
Why you need bash here? Why not just `sudo ./.github/heroku_install.sh`?

Reply:
this does not work, I have tried this:https://github.com/commercetools/commercetools-payment-integration-java/runs/4738268984?check_suite_focus=true#step:7:14

Reply:
You sure this is the problem? It seems like you didn't do `sudo chmod u+x ./.github/heroku_install.sh` before executing because the file does not have execute permission from what I see.

Reply:
your right bash is not needed anymore, thanks -> bc8702d